src/sentry/static/sentry/app/components/group/externalIssues.jsx

```diff
@@ -51,6 +51,57 @@ class ExternalIssueForm extends AsyncComponent {
     });
   };
 
+  onFieldChange = (label, value) => {
+    let {integrationDetails} = this.state;
+    let {action, group, integration} = this.props;
+    let config = integrationDetails[`${action}IssueConfig`];
+    let dynamicFields = new Set(
+      config
+        .filter(field => {
+          return field.updatesForm;
+        })
+        .map(field => {
+          return field.name;
+        })
+    );
+    if (dynamicFields.has(label)) {
+      let dynamicFieldValues = this.state.dynamicFieldValues || {};
+      dynamicFieldValues[label] = value;
```

Comment:
this whole function is pretty nested - could you put some part of it into it's own method? like "refetchConfig"?